Resources/Audit/qml/auditSelection.qml

```diff
@@ -256,9 +256,38 @@ Form {
 					}
 				}
 			}
+
+			GroupBox
+			{
+				id: 											samplingPlots
+				title: 										qsTr("Plots")
+
+				CheckBox { 
+					text: qsTr("Population and sample histograms")
+					name: "plotHistograms"
+					enabled: recordNumberVariable.count > 0	& monetaryVariable.count > 0				
+				}
+
+			}
 		}
 	}
 
+	CheckBox 
+	{ 
+			id: addSampleIndicator  
+			name: "addSampleIndicator"
+			text: qsTr("Add selection counter to data")
+			enabled: recordNumberVariable.count > 0
+
+			ComputedColumnField 
+			{ 
+					name: 			"sampleIndicatorColumn"
+					text: 			qsTr("Column name: ")
+					fieldWidth: 120
+					visible:    addSampleIndicator.checked
+			}
+	}
+
 	Item
 	{
 		Layout.preferredHeight: 			downloadReportSelection.height
```

Comment:
You need to update the jasp_nl.po for these changes also. (I can do this for you or tell you how if you don't know)

Reply:
Yes, would be nice if you could tell me so that I know for future changes!